### PR TITLE
edwardmack: retain at rank II

### PR DESCRIPTION
**Pull request by edwardmack**
Evidence to support my retention at rank 2.

### CODE REVIEW

File: evidence/edwardmack/0001-edwardmack-09-24-retention-evidence.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+| **Report Date** | Date of submission (2024/09/25) |
+| **Submitted by**| Edward Mack                     |
+
+
+## Member details
+
+- Matrix username: @mackcom:matrix.org
+- Polkadot address: 1ZYq7qWxvqC1rE4PKnTWwoVJYLtn99p9JM9cUn2WqbWr3Jt
+- Current rank: 2
+- Date of initial induction:
```
Comment (anaelleltd) [inline evidence/edwardmack/0001-edwardmack-09-24-retention-evidence.md:14]: This should read "Seeding".

File: evidence/edwardmack/0001-edwardmack-09-24-retention-evidence.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- Area(s) of Expertise/Interest: Gossamer node, parachain subsystems 
+
+
+## Reporting period
+
+- Start date: 2024/07/01
+- End date: 2024/09/25
+
+
+## Evidence
```
Comment (anaelleltd) [inline evidence/edwardmack/0001-edwardmack-09-24-retention-evidence.md:25]: Please make sure your adhere to the recommendations from the latest template:
"Explain why your contributions in relation to the Polkadot SDK are worthy of retention/promotion. Refer to the terms in Section 6 of the [Manifesto](https://github.com/polkadot-fellows/manifesto/blob/main/manifesto.pdf) and provide links to relevant content (i.e code, articles, media, etc.) to show that you are meeting all the requirements."

This section should consist mainly of a written argument, not a list of PRs, as advised here: https://hackmd.io/@polkadot/guidelines

File: evidence/edwardmack/0001-edwardmack-09-24-retention-evidence.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+| Gossamer node, candidate validation subsystem | Confirm invalidity errors are implemented in candidate validation   | [ChainSafe PR 4163](https://github.com/ChainSafe/gossamer/pull/4163) |  |
+
+
+## Voting record
+*Provide your voting record in relation to required thresholds for your rank.* 
+
+|  Ranks | Activity thresholds | Agreement thresholds | Member's voting activities                                                                                                                                                                                                                                                                                               | Comments |
+|---|---|---|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-|
+|III|70%   |100%  | I have voted on 0 out of 2 referenda in which I was eligible to vote (i.e 0 % voting activity). I intend to be more active regarding voting in the future.  | |
```
Comment (anaelleltd) [inline evidence/edwardmack/0001-edwardmack-09-24-retention-evidence.md:43]: Please do not override the example. Instead, you need to fill the row of the table that corresponds to your rank.
See an example of how this should be done below:
image

File: evidence/edwardmack/0001-edwardmack-09-24-retention-evidence.md (hunk 1 above)
Comment (edwardmack) [reply evidence/edwardmack/0001-edwardmack-09-24-retention-evidence.md]: I've updated this.

File: evidence/edwardmack/0001-edwardmack-09-24-retention-evidence.md (hunk 2 above)
Comment (edwardmack) [reply evidence/edwardmack/0001-edwardmack-09-24-retention-evidence.md:25]: Thank you for the comment and the link highlighting the guidelines, I've updated this.

File: evidence/edwardmack/0001-edwardmack-09-24-retention-evidence.md (hunk 3 above)
Comment (edwardmack) [reply evidence/edwardmack/0001-edwardmack-09-24-retention-evidence.md]: I've updated this, thanks for providing an example.

**Pull request merged**